lib/ims/lti/models/content_items/content_item.rb

```diff
@@ -19,12 +19,12 @@ def initialize(attributes = {})
     def self.from_json(json)
       data = json.is_a?(String) ? JSON.parse(json) : json
       case data['@type']
-        when 'ContentItem'
-          ContentItem.new.from_json(data)
-        when 'FileItem'
-          FileItem.new.from_json(data)
-        when 'LtiLinkItem'
-          LtiLinkItem.new.from_json(data)
+      when 'FileItem'
+        FileItem.new.from_json(data)
+      when 'LtiLinkItem', 'LtiLink'
+        LtiLinkItem.new.from_json(data)
+      else
```

Comment:
It's possible, but I think unlikely.  The previous code would have masked bad content-item JSON, or extended content-item JSON. I think the likely hood and ramifications are small enough that it's worth making the change.